### PR TITLE
Updated goals for all 25 participants

### PR DESCRIPTION
**Pull request by remware**

### CODE REVIEW

File: dc/helifuzzy.dc [hunk 1]
```diff
@@ -100,9 +103,7 @@ layer helifuzzy is
 	 end;
 	 
 	 -- Adherence as a function of progress
-	 function computeAdherance(w:World; p:Patient) : real is	 
-		 -- return p.nmsg_sent / w.day;   		 
-		 -- return weightProgress(p) *  stepsProgress(p) * fruitsProgress(p);
+	 function computeAdherance(w:World; p:Patient) : real is	  		 
 		 return (p.goal.w.v1*weightProgress(p) +  p.goal.w.v2*stepsProgress(p) + p.goal.w.v3*fruitsProgress(p))/(p.goal.w.v1 + p.goal.w.v2 + p.goal.w.v3) ;
 	 end;
```
Comment (ldiago) [inline dc/helifuzzy.dc:111]: Adh = (w1 *p1 + w2*p2 +w3*p3 + w4*p4 +w5*p5) / (w1 + w2 + w3 + w4 + w5)

File: dc/helifuzzy.dc [hunk 2]
```diff
@@ -131,7 +132,7 @@ layer helifuzzy is
 	     ||p.t_registered := w.day
 		 ||p.t_last_activity := w.day
 	     ||p.m:=1.0
-		 --||p.goal:= Goal( task -> control_weight(), target := Real3(v1:=71.0, v2:=0.0, v3:=0.0), w:= Real3(v1:=1.0, v2:=0.0, v3:=0))
+		 --||p.goal:= Goal( task -> control_weight(), target := RealVector(v1:=71.0, v2:=0.0, v3:=0.0), w:= RealVector(v1:=1.0, v2:=0.0, v3:=0))
          ||p.status -> hapa_motivation();
```
Comment (ldiago) [inline dc/helifuzzy.dc]: Do you want to change the goal during execution?

**Review by ldiago — CHANGES_REQUESTED** [review]
The formula for adh should be changed to Adh = (w1 *p1 + w2*p2 +w3*p3 + w4*p4 +w5*p5) / (w1 + w2 + w3 + w4 + w5)

File: dc/helifuzzy.dc (hunk 2 above)
Comment (remware) [reply dc/helifuzzy.dc]: No, there is no need to change the goal during execution. This line is commented out

**Pull request merged**